### PR TITLE
FS-656 SCA Performance Test Scripts

### PR DESCRIPTION
**Pull request by Rakesh-Eluri**

### CODE REVIEW

File: .bsp/sbt.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.8","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/rakesheluri/Library/Application Support/JetBrains/IdeaIC2022.2/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/rakesheluri/Library/Application%20Support/JetBrains/IdeaIC2022.2/plugins/Scala/launcher/sbt-launch.jar"]}
```
Comment (yangwong9664) [inline .bsp/sbt.json]: Can you please add ./bsp to the .gitignore, so this file doesn't get uploaded to git

File: repository.yaml [hunk 2]
```diff
@@ -1 +1,2 @@
-repoVisibility: public_0C3F0CE3E6E6448FAD341E7BFA50FCD333E06A20CFF05FCACE61154DDBBADF71
\ No newline at end of file
+repoVisibility: public_0C3F0CE3E6E6448FAD341E7BFA50FCD333E06A20CFF05FCACE61154DDBBADF71
+digital-service: single-customer-account-frontend
```
Comment (yangwong9664) [inline repository.yaml]: Why has this been added?

File: src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourney.scala [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      .check(status.in(200))
+      .check(currentLocation.is(YourTaxesAndBenefits + "/single-customer-account/taxes-and-benefits": String))
+
+  def getMessages: HttpRequestBuilder =
+    http("GET - SCA Messages")
+      .get(Messages + "/single-customer-account/messages": String)
+      .check(status.in(200))
+      .check(currentLocation.is(Messages + "/single-customer-account/messages": String))
+
+  def getFeedbackPage: HttpRequestBuilder =
```
Comment (yangwong9664) [inline src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourney.scala]: we don't need to perf test the feedback service

File: src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourneySimulation.scala [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+class SCAUserJourneySimulation extends PerformanceTestRunner {
+
+  setup("SCA-HomePage", "SCA Home Page") withRequests(
+    getGGStartPage,
+    postGGCredentials(nino = "AA999999A", isLocal = runLocal),
+    getSCAHomePage,
+    getYourTaxesAndBenefits,
+    getMessages,
+    getFeedbackPage
```
Comment (yangwong9664) [inline src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourneySimulation.scala]: same as above

**Review by yangwong9664 — CHANGES_REQUESTED** [review]

File: repository.yaml (hunk 2 above)
Comment (Rakesh-Eluri) [reply repository.yaml]: This I believe is created as part of Jenkins build but however now removed below:
digital-service: single-customer-account-frontend

File: .bsp/sbt.json (hunk 1 above)
Comment (Rakesh-Eluri) [reply .bsp/sbt.json]: Sure will do.

File: src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourneySimulation.scala (hunk 4 above)
Comment (Rakesh-Eluri) [reply src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourneySimulation.scala]: Removed Feedback service from script.

File: src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourney.scala (hunk 3 above)
Comment (Rakesh-Eluri) [reply src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourney.scala]: Removed Feedback service from script.

File: .bsp/sbt.json (hunk 1 above)
Comment (Rakesh-Eluri) [reply .bsp/sbt.json]: Actioned.

File: repository.yaml (hunk 2 above)
Comment (Rakesh-Eluri) [reply repository.yaml]: Actioned

File: src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourney.scala (hunk 3 above)
Comment (Rakesh-Eluri) [reply src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourney.scala]: Actioned

File: src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourneySimulation.scala (hunk 4 above)
Comment (Rakesh-Eluri) [reply src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourneySimulation.scala]: Actioned

**Review by yangwong9664 — APPROVED** [review]

**Pull request merged**